railties/lib/rails/generators/rails/authentication/templates/views/passwords_mailer/reset.text.erb

```diff
@@ -0,0 +1,2 @@
+You can reset your password within the next 15 minutes on this password reset page:
+<%%= edit_password_url(@user.password_reset_token) %>
```

Comment:
Where is the `password_reset_token` supposed to be coming from? It is not generated in the migration, and not included in `has_secure_password`, as far as I can tell... And I think it would need to be generated in the `Passwords#create` action (with a validity of 15 minutes, which would probably require another database column).

Reply:
It is generated by has_secure_password now, yes. See https://github.com/rails/rails/blob/main/activemodel/lib/active_model/secure_password.rb#L224

Reply:
Oh, I only looked in the docs for 7.2 and therefore did not see it, my bad 🙈 Thanks for the pointer - it's cool that no database field is needed at all, but everything is baked into the token 👌